packages/workflow/docs/Workflow.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { Button } from "@kaizen/button"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import { Workflow, WorkflowExit, WorkflowProps } from "../"
+
+const defaultArgs = {
+  workflowName: "Create a self-reflection cycle",
+  stepName: "Settings",
+  steps: ["Settings", "Questions", "Preview", "Employees", "Schedule"],
+  isComplete: false,
+  status: {
+    content: "Draft",
+    vairant: "statusDraft",
+  },
+  previousAction: <Button reversed label="back" />,
+  nextAction: <Button reversed label="next" />,
+  actions: [
+    <WorkflowExit
+      key={"would-use-uui"}
+      exitLabel="Save and close"
+      exitTitle="Before you exit"
+      exitDescription="Your content has not yet been saved. Click the button below or discard the changes"
+      confirmExitLabel="Close and save"
+      dismissExitLabel="Dismiss"
+      onExit={(): void => alert("mock example of a save action")}
+    />,
+  ],
+} satisfies WorkflowProps
+
+const meta = {
+  tags: ["autodocs"],
+  title: "Workflow",
+  component: Workflow,
+  parameters: {
+    docs: {
+      source: { type: "code" },
+      container: ComponentDocsTemplate,
+    },
+    installation: [
+      "npm install @kaizen/workflow",
```

Comment:
```suggestion
      "yarn add @kaizen/workflow",
```

I think everyone uses `yarn`?

Reply:
Except for those using pnpm 😉 pretty sure yep. I think this was generated from the plop template automatically so complete glazed over it - should props update that too 👍